topo/topo_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package topo
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	tfake "github.com/google/kne/api/clientset/v1beta1/fake"
+	tpb "github.com/google/kne/proto/topo"
+	"github.com/h-fam/errdiff"
+	kfake "k8s.io/client-go/kubernetes/fake"
+
+	"k8s.io/client-go/rest"
+)
+
+var (
+	opts []Option
+)
+
+func TestCreateDeletTopology(t *testing.T) {
```

Comment:
Better to break this into two test funcs for clarity:

func TestCreateTopology() {}

func TestDeleteTopology() {}